lab2/app.js

```diff
@@ -0,0 +1,11 @@
+
+var x = window.incrementalGame.state.counter;
+var y = document.getElementById("numb")+1;
+window.onload = function()
+  {
+    document.querySelector('#pick').onclick = function ()
+    {
+      console.log(x++);
```

Comment:
This statement `x++` does not increment the window.incrementalGame.state.counter